### PR TITLE
fix: Add length for mysql index

### PR DESCRIPTION
**Pull request by catto**
I found the same issue as screwdriver-cd/screwdriver#325 on the latest version of Screwdriver.
I added length limitations for each key to fix it.

### CODE REVIEW

File: models/event.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ const MODEL = {
     createTime: Joi
         .string()
         .isoDate()
+        .max(32)
```
Comment (catto) [inline models/event.js]: Should `createTime` be date, not string?

Comment (stjohnjohnson) [inline models/event.js]: Good question.  I think we were storing it as a String and letting the users parse it.  But we may want to switch to Date.

Comment (catto) [inline models/event.js]: I got it 🐈

**Comment by tkyi** [comment]
We're going to wait to merge this PR. Might have to go through and rerun a bunch of other pipelines to pick up the new `data-schema` version with your change after it's merged.

**Pull request merged**